contracts/FlowIDTableStaking.cdc

```diff
@@ -645,17 +645,25 @@ pub contract FlowIDTableStaking {
             FlowIDTableStaking.stakingEnabled = true
         }
 
+        /// Ends the staking Auction by removing any unapproved nodes
+        /// and setting stakingEnabled to false
+        pub fun endStakingAuction(approvedNodeIDs: {String: Bool}) {
```

Comment:
Will this be called by the `EpochLifecycle` contract?

Reply:
yeah we can call it from the lifecycle contract